shanoir-ng-studies/src/main/java/org/shanoir/ng/tag/model/StudyTag.java

```diff
@@ -0,0 +1,88 @@
+package org.shanoir.ng.tag.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+import org.shanoir.ng.shared.hateoas.HalEntity;
+import org.shanoir.ng.study.model.Study;
+import org.shanoir.ng.subjectstudy.model.SubjectStudyTag;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+public class StudyTag extends HalEntity {
```

Comment:
As you are adding a new entity, you need to create a  corresponding database script to add the table. (you can make a "show create table" or something like this)